### DISCUSSION TITLE
request consolidate various forks with dev work in them -some stale now

### DISCUSSION THREAD

**Discussion by GoZippy** [coinfoundry/miningcore#920] [General]
Has anyone reached out to the devs that forked and added but did not request to merge back up?

https://github.com/minernl/miningcore

for example has given some major work to same project adding 90 coins... or more.. said he never got but 0.0001 ltc as dev reward so he stopped releasing and dev work..

Perhaps merge up is in order so we call can see changes and give credit where due - not reinvent what is already working...

**Comment by oliverw** [comment]
I've already merged some of the functionality from that fork back into the main codebase. Not all of the changes were usable.

**Reply by GoZippy** [reply]
ok cool - sorry for such noob here - I am learning but having fun... eventually I will jump into code more and help if ever asked...

**Reply by BrianBMN** [reply]
Don't wait! Fork the repo, add your updates, and then submit a PR :D

**Comment by BrianBMN** [comment]
I was attracted to the MinerNL fork because it had support for so many coins. However, after digging into it looks as though the added algos were either not compiled into or not compiled correctly into libmultihash.dll. So things like VerusCoin were throwing an error of "unsupported coin." But now that the main branch is active again, I'll focus here.

**Reply by GoZippy** [reply]
close this - but I suggest we keep eye on forks to see what good things are implemented we can wrap back into base since many forks are individuals that never submit pull requests back up. Or - keep this thread open so anyone that sees good idea post link here and we can create own fork and submit pr back up
